mcstatus/server.py

```diff
@@ -59,6 +59,18 @@ class JavaServer(MCServer):
 
     DEFAULT_PORT = 25565
 
+    def __init__(self, host: str, port: int | None = None, timeout: float = 3, query_port: int | None = None):
+        """
+        :param host: The host/ip of the minecraft server.
+        :param port: The port that the server is on.
+        :param timeout: The timeout in seconds before failing to connect.
+        :param query_port: Typically the same as ``port`` but can be different.
+        """
+        super().__init__(host, port, timeout)
+        if query_port is None:
+            query_port = self.DEFAULT_PORT
```

Comment:
```suggestion
            query_port = port or self.DEFAULT_PORT
```

Reply:
Good catch! This is why I love having code reviewed. Thanks.